README.md

```diff
@@ -1,39 +1,41 @@
 # yay
 
-Yet another Yogurt - An AUR Helper written in Go
+Yet Another Yogurt - An AUR Helper Written in Go
 
 #### Packages
 
 [![yay](https://img.shields.io/aur/version/yay.svg?label=yay)](https://aur.archlinux.org/packages/yay/) [![yay-bin](https://img.shields.io/aur/version/yay-bin.svg?label=yay-bin)](https://aur.archlinux.org/packages/yay-bin/) [![yay-git](https://img.shields.io/aur/version/yay-git.svg?label=yay-git)](https://aur.archlinux.org/packages/yay-git/) [![GitHub license](https://img.shields.io/github/license/jguer/yay.svg)](https://github.com/Jguer/yay/blob/master/LICENSE)  
+
+## Objectives
+
 There's a point in everyone's life when you feel the need to write an AUR helper because there are only about 20 of them.
 So say hi to 20+1.
 
-Yay was created with a few objectives in mind and based on the design of [yaourt](https://github.com/archlinuxfr/yaourt), [apacman](https://github.com/oshazard/apacman) and [pacaur](https://github.com/rmarquis/pacaur):
+_Yay_ is based on the design of [yaourt](https://github.com/archlinuxfr/yaourt), [apacman](https://github.com/oshazard/apacman) and [pacaur](https://github.com/rmarquis/pacaur). It is developed with these objectives in mind:
 
-* Have almost no dependencies
-* Provide an interface for pacman
-* Have yaourt-like search
+* Wrap the _pacman_ interface
```

Comment:
This doesn't sit right with me. The interface is how the users interact with it. We can either mimic the interface (which is what we do for AUR packages) or wrap pacman itself (which we also do for most other commands -D -Q ect.) You can't really wrap an interface.

Reply:
A good technical point. Reverted to original wording.